src/common/id/UUID.h

```diff
@@ -0,0 +1,25 @@
+/* Copyright (c) 2021 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License.
+ */
+
+#include <boost/uuid/uuid.hpp>             // uuid class
+#include <boost/uuid/uuid_generators.hpp>  // generators
+#include <boost/uuid/uuid_io.hpp>          // streaming operators etc
+
+namespace nebula {
+
+class UUID {
+ public:
+  static boost::uuids::uuid getId() {
+    boost::uuids::uuid uuid = boost::uuids::random_generator()();
```

Comment:
As I known, the random generated uuid contains probability of collision. Do you consider it?